packages/cli/test/integration/commands/license.cmd.test.ts

```diff
@@ -0,0 +1,32 @@
+import { InternalHooks } from '@/InternalHooks';
+import { License } from '@/License';
+import { LoadNodesAndCredentials } from '@/LoadNodesAndCredentials';
+import { ClearLicenseCommand } from '@/commands/license/clear';
+import { Config } from '@oclif/core';
+import Container from 'typedi';
+import { mockInstance } from '../../shared/mocking';
+
+const oclifConfig = new Config({ root: __dirname });
+
+beforeAll(async () => {
+	mockInstance(InternalHooks);
+	mockInstance(LoadNodesAndCredentials);
+	await oclifConfig.load();
+});
+
+test('license:clear invokes shutdown() to release any floating entitlements', async () => {
+	const cmd = new ClearLicenseCommand([], oclifConfig);
+	await cmd.init();
+
+	const license = Container.get(License);
+
+	jest.spyOn(license, 'init');
+	jest.spyOn(license, 'shutdown');
```

Comment:
Let's mock the license instead of using the real one?

```suggestion
	const license = mockInstance(License);
```

Reply:
makes sense! Thanks for the suggestion!